highfidelity/hifi-content
# removed unused invisible sphere asset, updated spawner to not spam log

## Test Plan
1. Open up the script log
2. Go to the cafe in hifi://limitless
3. Check that the log isn't getting spammed with `Found script in cache: "https://hifi-content.s3.amazonaws.com/DomainContent/Hub-staging/food/scripts/crunch.js"`
4. In HMD, grab a food item and eat it
5. Check that the log only added one more "crunch.js" print
